pkgs/by-name/ko/komac/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, darwin
+, testers
+, komac
+}:
+
+let
+  version = "2.1.0";
+  src = fetchFromGitHub {
+    owner = "russellbanks";
+    repo = "Komac";
+    rev = "v${version}";
+    hash = "sha256-L8UYpNqjRyqf4hPQwD9LaXWu6jYaP34yTwTxcqg+e2U=";
+  };
+in
+rustPlatform.buildRustPackage {
+  inherit version src;
+
+  pname = "komac";
+
+  cargoHash = "sha256-J4QZzbyDr4SDt6LlAy9ZdpqgIufZCZHmOC9eu70wMsM=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk_11_0.frameworks.SystemConfiguration
+  ];
+
+  passthru.tests.version = testers.testVersion {
+    inherit version;
+
+    package = komac;
+    command = "komac --version";
+  };
+
+  meta = with lib; {
+    description = "The Community Manifest Creator for WinGet";
+    homepage = "https://github.com/russellbanks/Komac";
+    changelog = "https://github.com/russellbanks/Komac/releases/tag/${src.rev}";
+    license = licenses.gpl3;
```

https://github.com/russellbanks/Komac?tab=readme-ov-file#license


```suggestion
    license = licenses.gpl3Plus;
```

Don't use `licenses.gpl3`

---

I didn't notice gpl3 is deprecated specifier https://github.com/NixOS/nixpkgs/commit/9d5bd8b835ddd62db5857b86fcda5af00c237bfa and komac clarified to allow later versions 🙇‍♂️ 

Updated in https://github.com/NixOS/nixpkgs/pull/296350/commits/aa5635a987033c122ab055f67036d88ea6e759a8

(Should I squash into 1 `komac: init` commit?)